packages/terra-demographics-banner/src/DemographicsBanner.module.scss

```diff
@@ -88,6 +89,7 @@
     flex: 1;
     margin-right: auto;
     margin-top: var(--terra-demographics-banner-application-content-margin-top);
+    padding-bottom: 0.2rem;
```

Comment:
should `padding-bottom` be themeable here...? @neilpfeiffer @jeremyfuksa

Reply:
Based on the way the code is constructed, I'm inclined to say no. Especially since it works with all themes currently available.